components/strapi-sections/strapi-components/title-and-body.js

```diff
@@ -1,32 +1,9 @@
-// import PropTypes from "prop-types";
-import { Box, Typography } from "@mui/material";
-import style from "./strapi-components.module.css";
-import Markdown from "react-markdown";
+import { Section } from '../../layout';
 
 export const TitleAndBody = ({ data }) => {
-  return (
-    <>
-      {data.sectionContent.map((section) => {
-        return (
-          <Box className={style.section} key={section.id + section.leftTitle}>
-            {section.leftTitle && (
-              <Box className={style.titleContainer}>
-                <Typography variant="h3" className={style.title}>
-                  {section.leftTitle}
-                </Typography>
-              </Box>
-            )}
-            <Box className={style.content}>
-              <Markdown linkTarget="_blank">{section.rightBody}</Markdown>
-            </Box>
-          </Box>
-        );
-      })}
-    </>
-  );
-};
-
-// Section.propTypes = {
-//   title: PropTypes.string,
-//   children: PropTypes.node.isRequired,
-// };
+  return data.sectionContent.map((section) => (
+    <Section title={ section.leftTitle } key={section.id + section.leftTitle}>
+      { section.rightBody }
```

Comment:
I was going to suggest using the Markdown component here from react-markdown, but I just saw that you're already working on creating the structure for a custom Markdown component, so we can incorporate the custom one here afterwards!